connector/src/it/scala/com/datastax/spark/connector/doc/DocCheck.scala

```diff
@@ -5,19 +5,20 @@ import com.datastax.spark.connector.util.{ConfigCheck, RefBuilder}
 import java.io.File
 
 import com.datastax.spark.connector.cluster.DefaultCluster
+import org.scalatest.{FlatSpec, Matchers}
 
-class DocCheck extends SparkCassandraITFlatSpecBase with DefaultCluster {
+class DocCheck extends FlatSpec with Matchers {
```

Comment:
Probably more of a question than a comment: can you elaborate on the dropping of DefaultCluster here?  I can't see any relationship between what it provides and your changes here, although it's entirely possible I'm missing something obvious.

Reply:
The "DefaultCluster" means that it starts up a C* CCM Cluster in this test suite, since this test suite doesn't actually use a C* cluster I removed it so I could run the test faster ;)